hail/python/hail/docs/change_log.md

```diff
@@ -20,10 +20,25 @@ such in the reference documentation, which may change at any time**.
 
 Please note that **forward compatibility should not be expected, especially
 relating to file formats**: this means that it may not be possible to use
-an earlier version of Hail to read files written in a later version.  
+an earlier version of Hail to read files written in a later version.
 
 ---
 
+## Version 0.2.26
+
```

Comment:
still needs date

Reply:
Why? This PR just updates the changelog. I feel like we really should have individual PRs update the change log incrementally, finally updating the released date when the version is finally released.

Reply:
For this PR, if we tag this exact commit after it's merged, then it's fine for me to update the date here. Otherwise it just feels premature.

Reply:
> I feel like we really should have individual PRs update the change log incrementally

This doesn't work super well because of merge conflicts.

> finally updating the released date when the version is finally released.

That's what this is. We release the commits that update the changelog and the patch version, like this one.

Reply:
> > I feel like we really should have individual PRs update the change log incrementally
>
> This doesn't work super well because of merge conflicts.

I'm unconvinced. Merge conflicts, especially here are easy to resolve and a fact of life when working the way we do.

Reply:
> a fact of life when working the way we do.

I don't know of any files that we're all appending to every few commits... that's the problem here.

There's also the issue that a commit can't update the changelog with its PR number before that PR is created!